src/Components/Facility/ConsultationDetails.tsx

```diff
@@ -171,7 +171,7 @@ export const ConsultationDetails = (props: any) => {
       <div className="px-2 pb-2">
         <nav className="flex justify-between flex-wrap">
           <PageTitle
-            title="Patient Dashboard"
+            title="Patient Details"
```

Comment:
@rithviknishad Was this unintended

Reply:
It was unintended action.
After looking into: https://github.com/coronasafe/care_fe/pull/2381/commits/a34d7821b4b41f2e2f6705200e0015da0eab84df and https://github.com/coronasafe/care_fe/pull/2381/commits/845a268db201a9c3b6a83b9715ca9675be81ee95, seems like intended action was to remove that commit from this PR since it was not related to this, but didn't expect this would happen. My bad. Resolved again in #2754